libraries/botbuilder-ai/tests/qnaMaker.test.js

```diff
@@ -18,29 +20,45 @@ class TestContext extends TurnContext {
     }
 }
 
-xdescribe('QnAMaker', function () {
+describe('QnAMaker', function () {
     this.timeout(10000);
+    const testFiles = fs.readdirSync(`${ __dirname }/TestData/${ this.title }/`);
+    const mockQnA = true;
```

Comment:
Move this const above the keys section for readability ([see this sample](https://github.com/Microsoft/botbuilder-js/blob/master/libraries/botbuilder-ai/tests/luisRecognizer.test.js#L11-L13)).

Reply:
I assumed you were talking about `mockQnA` and left `testFiles` where it is since that variable needs the title description of the tests.